app/form_models/jobseekers/job_application/employment_history_form.rb

```diff
@@ -14,27 +14,37 @@ def storable_fields
         end
 
         def unstorable_fields
-          %i[unexplained_employment_gaps_present employment_history_section_completed]
+          %i[unexplained_employment_gaps_present employment_history_section_completed employments]
         end
 
         def load_form(model)
-          super.merge(unexplained_employment_gaps: model.unexplained_employment_gaps).merge(completed_attrs(model, :employment_history))
+          super.merge(employments: model.employments,
+                      unexplained_employment_gaps: model.unexplained_employment_gaps)
+               .merge(completed_attrs(model, :employment_history))
         end
       end
-      attr_accessor(*FIELDS, :unexplained_employment_gaps)
+      attr_accessor(*FIELDS, :unexplained_employment_gaps, :employments)
 
-      validate :employment_history_gaps_are_explained
+      validate :employment_history_gaps_are_explained, if: -> { unexplained_employment_gaps_present == "true" && employment_history_section_completed }
```

Comment:
I thought we had migrated field like this `unexplained_employment_gaps_present` to boolean?

Reply:
I've just managed to remove it - it was making failing scenarios pass as well by being nil in quite a few places